src/libs/ReportActionsUtils.ts

```diff
@@ -269,7 +269,7 @@ function getContinuousReportActionChain(sortedReportActions: ReportAction[], id?
     }
 
     if (index === -1) {
-        return [];
+        return id ? [] : sortedReportActions;
```

Comment:
@nkdengineer can you add a comment here?

```
// if no non-pending action is found, that means all actions on the report are optimistic
// in this case, we'll assume the whole chain of reportActions is continuous and return it in its entirety
```

Reply:
@roryabraham I updated.